### ISSUE TITLE
Can't scribe scrolls

### ISSUE THREAD

**Issue by grannypron** [grannypron/uaf_levels#81]
First attempt at scribing a scroll results in the message "Un-implemented code: 04D67AB9"

Subsequent attempts result in no action. A # sign appears next to the item name in the inventory screen, but that is it.  Resting for any number of days does not add the spell to the PCs spellbook.

**Comment by grannypron** [comment]
The # sign is because the item is out of charges.  That seems to be working relatively well, although in PoR, the item disappears from the inventory when it is out of charges.

**Comment by grannypron** [comment]
When I remove the NotImplemented() call with a simple implementation that calls KnowSpellyyy(), the scribe works pretty reasonably.

```

void SCRIBE_MENU_DATA::OnReturnToTopOfQueue(void)
{
  if (m_itemID_scroll.IsValidItem())
  {
    ITEM_DATA *pScroll;
    pScroll = itemData.GetItem(m_itemID_scroll);
    ASSERT (pScroll->spellID.IsValidSpell());
    if (pScroll->spellID.IsValidSpell())
    {
      SPELL_DATA *pSpell = spellData.GetSpell(pScroll->spellID);
      m_pDude->KnowSpellyyy(pSpell, true);
    };
  };
  PopEvent();
}

```

**Comment by grannypron** [comment]
Filed at https://github.com/grannypron/uaf/issues/40 and being reviewed at http://ua.reonis.com/index.php?topic=4032.0

**Comment by grannypron** [comment]
Closed with upgrade to 5.27
